File: backend/api/src/main/java/edu/miu/cs425/flightbookingsystem/dto/FlightRouteDTO.java
package edu.miu.cs425.flightbookingsystem.dto;

import java.time.LocalTime;

public record FlightRouteDTO(
        Long id,
        String departureCity,
        String arrivalCity,
        String flightNumber,
        LocalTime departureTime,
        LocalTime arrivalTime) {
}

File: frontend/flight-booking/src/model/FlightRouteDTO.ts
import { FlightScheduleDTO } from './FlightScheduleDTO';

export interface FlightRouteDTO {
  id?: number | null;
  departureCity: string;
  arrivalCity: string;
  flightNumber: string;
  departureTime: string;
  arrivalTime: string;
  flightScheduleDTOS: FlightScheduleDTO[];
}
